GPU/GPUTracking/Standalone/CMakeLists.txt

```diff
@@ -50,7 +50,7 @@ set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
 if(BUILD_DEBUG)
   set(CMAKE_CXX_FLAGS "-O0 -ggdb") # -fsanitize=address,undefined -fno-sanitize=vptr
-  set(CMAKE_BUILD_TYPE DEBUG)
+  set(CMAKE_BUILD_TYPE Debug)
```

Comment:
Could you please not change the settings for the standalone build. Anyway not part of O2. And the alidist build type is also DEBUG / RELEASE, so no reason to enforce camel-case.